test-schema.json

```diff
@@ -56,7 +63,14 @@
                     "type": "string",
                     "format": "uri"
                 },
-                "description": { "type": "string" },
+                "description": {
+                    "description": "The test description",
+                    "type": "string"
+                },
+                "comment": {
```

Comment:
Isn't it $comment

Reply:
Doesn't have to be. The "$" isn't reserved or anything. I'm on the fence. What's your preference? (I'll change the test schema to match.) On one hand, I like "$" in schemas and "comment" is just an ordinary property. On the other, would "$" make the property stand out? Hmm.

Reply:
It's true you can use whatever, but it actually *is* [reserved](https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.8.3).

(Not in the draft we're using, but in the current ones)

Reply:
Or, sorry I didn't pay attention close enough, the comments here we're putting not in schemas right? Just in the top-level test case? In that case yeah, not `$comment`, it's not a schema. Never mind :).

lgtm